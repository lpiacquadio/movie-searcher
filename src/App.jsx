import React from 'react'
import { BrowserRouter, Routes, Route } from 'react-router-dom'

import { Home } from './pages/Home'
import { Detail } from './pages/Detail'

export function App() {
    return (
        <BrowserRouter>
            <Routes>
                <Route index path="/" element={<Home />} />
                <Route index path="/:id" element={<Detail />} />
            </Routes>
        </BrowserRouter>
    )
}
