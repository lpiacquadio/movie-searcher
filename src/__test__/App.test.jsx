import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, test } from 'vitest'
import { App } from '../App'

describe('App', () => {
    test('App component renders correctly', () => {
        const component = renderer.create(<App />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
