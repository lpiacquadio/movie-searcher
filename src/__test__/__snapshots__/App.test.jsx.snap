// Vitest Snapshot v1

exports[`App > App component renders correctly 1`] = `
<div>
  Home Page
</div>
`;
